src/public/js/util.js

```diff
@@ -38,6 +38,12 @@ var UTIL = (function() {
      0xb4, 0x7c, 0xb8, 0x26, 0x77, 0x99, 0xe3, 0xa5,
      0x67, 0x4a, 0xed, 0xde, 0xc5, 0x31, 0xfe, 0x18,
      0x0d, 0x63, 0x8c, 0x80, 0xc0, 0xf7, 0x70, 0x07]);
+  log3.get = function(x) {
```

Comment:
And if you asked for `test.has`, which doesn't exist, you'd get an `undefined`

(and if you tried calling `test.has(...)` you'd get a `test.has is not a function`)